src/bentoml/_internal/frameworks/pytorch.py

```diff
@@ -75,9 +75,7 @@ def load_model(
 
     weight_file = bentoml_model.path_of(MODEL_FILENAME)
     with Path(weight_file).open("rb") as file:
-        model: "torch.nn.Module" = torch.load(
-            file, map_location=device_id, pickle_module=cloudpickle
-        )
+        model = torch.load(file, map_location=device_id)
```

Comment:
@bojiang I dont' think we can use cloudpickle here, since pytorch actually use `pickle_module.Unpickler` inside torch.load, which cloudpickle doesn' have a `Unpickler` module.

Reply:
@aarnphm which version of PyTorch are you using? It works in my test

Reply:
im using the latest version.